app/views/ops/_diagnostics_savedreports.html.haml

```diff
@@ -1,23 +1,25 @@
 = render :partial => "layouts/flash_msg"
-%h3= _("Report Results by User")
+= react('DiagnosticsOrphanedData', {:initialData => diagnostics_orphaned_data_list(@sb[:orphaned_records])})
+-# %h3= _("Report Results by User")
```

Comment:
Is there a reason the old code is being commented out here instead of just being removed?

Reply:
Hey @anu-jose-ibm, if you feel like the new changes work. please remove the old code.

Reply:
no nothing. removing that.